.github/workflows/cd.yml

```diff
@@ -0,0 +1,30 @@
+name: CD
+
+on: [release]
+
+jobs:
+  publish-chart:
+    runs-on: ubuntu-latest
+    needs: test-chart
```

Comment:
you're missing this step - copy it over?

Reply:
Yep, noticed this too and decided to remain with "push" but filter on `tags` only.